css-contain-1/annotated-2019-09-05.html

```diff
@@ -1941,7 +1941,7 @@ <h2 class="no-num no-ref heading settled" id="property-index"><span class="conte
       <th scope="col">%ages
       <th scope="col">Anim­ation type
       <th scope="col">Canonical order
-      <th scope="col">Com­puted value
```

Comment:
What actually changed in this line?

Reply:
There was a zero width space or some other odd character that was splitting the word silently